### PR TITLE
samples: openthread: Remove nrf52833 support for CLI and CoAP

### PR DESCRIPTION
**Pull request by maciejbaczmanski**
Support for `nrf52833dk_nrf52833` target in openthread CLI and CoAP samples has been withdrawn. Updated project files accordingly.

### CODE REVIEW

**Review by canisLupus1313 — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -465,6 +465,9 @@ Trusted Firmware-M (TF-M) samples
 Thread samples
 --------------
 
+* Removed support for ``nrf52833dk_nrf52833`` target in :ref:`ot_cli_sample`,
+  :ref:`coap_client_sample` and :ref:`coap_server_sample`.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:469]:
```suggestion
* Removed support for the ``nrf52833dk_nrf52833`` build target in :ref:`ot_cli_sample`, :ref:`coap_client_sample` and :ref:`coap_server_sample`.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by pjasinski990 — APPROVED** [review]
lgtm

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 1 above)
Comment (maciejbaczmanski) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst]: done

**Review by wiba-nordic — APPROVED** [review]
Looks good to me.

**Review by rlubos — APPROVED** [review]

**Pull request merged**